ffi/src/lib.rs

```diff
@@ -16,9 +16,19 @@ use delta_kernel::snapshot::Snapshot;
 use delta_kernel::{DeltaResult, Engine, Error, Table};
 use delta_kernel_ffi_macros::handle_descriptor;
 
+/// cbindgen:ignore
+#[cfg(feature = "developer-visibility")]
+pub mod handle;
+#[cfg(not(feature = "developer-visibility"))]
 pub(crate) mod handle;
```

Comment:
```suggestion
#[cfg_attr(feature = "developer-visibility", visibility::make(pub))]
#[cfg_attr(not(feature = "developer-visibility"), visibility::make(pub(crate)))]
mod handle;
```
```suggestion
#[cfg(feature = "developer-visibility")]
pub mod handle;
#[cfg(not(feature = "developer-visibility"))]
pub(crate) mod handle;
```

Reply:
I tried that first, but passing a non-inline module as input to proc macros is apparently unstable Rust...